### ISSUE TITLE
Consider adding the new Procedural Parts fork

### ISSUE THREAD

**Issue by HebaruSan** [KSP-CKAN/NetKAN#6308]
Procedural Parts has a new active fork. The author expressed interest in adding it to CKAN but frustration with the process:

- https://github.com/Tidal-Stream/ProceduralParts
- https://forum.kerbalspaceprogram.com/index.php?/topic/169250-131-procedural-parts-tidal-stream-branch/

@Tidal-Stream, there are a few options, all pretty easy:

- Switch the main ProceduralParts entry to look at your repository
- Release ProceduralPartsContinued as a new mod

I'm creating this issue to hash out what, if anything, we should do. And thanks for taking up maintenance of this mod, it's a dependency for one of mine and I've considered adopting it myself!

**Comment by HebaruSan** [comment]
My suggestion would be to update the existing netkan to point to this new repo. We can always change it back if we need to, and it would be better for dependencies. Plus it sounds like @Tidal-Stream's intent is to maintain this fork conservatively, with just enough changes to keep it running, so we're unlikely to end up with wildly divergent forks that can't be reconciled with one another if the previous branch resumes activity.

**Comment by HebaruSan** [comment]
Are you in active contact with @Starwaster? I don't think we technically need his sign-off, but it would definitely be a better way to proceed to keep everybody happy.

**Comment by HebaruSan** [comment]
@Tidal-Stream, FYI, I installed this on KSP 1.4.0 to do some other testing, and it seems to work fine. So you might be able to expand the compatible version range; please let us know if you want any help from CKAN with that.

**Comment by Tidal-Stream** [comment]
@HebaruSan I waited a bit after the release of 1.4.1 and recompiled it. The netkan configuration uses `"$vref" : "#/ckan/ksp-avc"` so it should automatically index the latest release in the next hours, right?

**Comment by HebaruSan** [comment]
Yup:  https://github.com/KSP-CKAN/CKAN-meta/commit/5f90d22453e44af4f5b4331c85d354401c293ae9
